elements/pl-code/pl-code.py

```diff
@@ -71,6 +73,16 @@ def render(element_html, element_index, data):
     elif len(code) > 0 and (code[0] == '\n' or code[0] == '\r'):
         code = code[1:]
 
+    if input_file_name is not None:
+        file_path = os.path.join(data['options']['question_path'], input_file_name)
```

Comment:
For consistency with elements like `<pl-figure>`, I'd consider supporting a `source-directory` to allow pulling in files from locations besides the current question.